webapp/app/components/message/message.component.js

```diff
@@ -0,0 +1,55 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Map } from 'immutable';
+import moment from 'moment';
+import { ListItem, ListItemText } from 'material-ui/List';
+
+import { Author, Data, MessageContent } from './message.styles';
+import { UserAvatar } from '../';
+
+
+export class Message extends PureComponent {
+  static propTypes = {
+    message: PropTypes.instanceOf(Map),
+  };
+
+  get time() {
+    const data = moment(this.props.message.get('publicationTime'));
+
+    if (moment().isSame(data, 'd')) {
+      return moment(data, 'hmm').format('HH:mm');
+    }
+    return moment(data, 'hmm').format('HH:mm DD.MM.YYYY');
+  }
+
+  get author() { return this.props.message.get('author'); }
+
+  get content() { return this.props.message.get('content'); }
+
+  get primaryText() {
```

Comment:
o tym już chyba Martin pisał w poprzedniej PRce. Jako codeStyle - jeśli funkcja zwraca komponent JSxowy to warto nazywać to `renderPrimaryText` zamiast gettera.